static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -506,6 +520,10 @@ function WidgetBuilder({
   }
 
   async function dataIsValid(widgetData: Widget): Promise<boolean> {
+    if (!isMounted.current) {
```

Comment:
we already have this condition in the handleSave, does it make sense to have it here too?